libc-top-half/musl/src/thread/wasm32/wasi_thread_start.s

```diff
@@ -0,0 +1,26 @@
+	.text
+
+	.export_name	wasi_thread_start, wasi_thread_start
+
+	.globaltype	__stack_pointer, i32
```

Comment:
In the long run we might want to make this `.S` rather than a `.s` so we can using macros like `__wasm64__` to detect pointer size. e.g.: https://github.com/emscripten-core/emscripten/blob/0d7014e1b20ead9739f1fce54b316152413601c3/system/lib/compiler-rt/stack_ops.S#L6-L12



For now this seems fine though.